packages/odf/components/create-storage-system/create-storage-system-steps/security-and-network-step/multus.tsx

```diff
@@ -80,35 +89,110 @@ export const MultusNetworkConfigurationComponent: React.FC<
   publicNetwork,
   networkType,
   systemNamespace,
+  isMultusAcknowledged,
+  setIsMultusAcknowledged,
 }) => {
   const { t } = useCustomTranslation();
+  const handleMultusToggle = (_event: any, checked: any) => {
+    setNetworkType(checked ? NetworkType.MULTUS : NetworkType.DEFAULT);
+    if (!checked) {
+      setIsMultusAcknowledged(false);
+    }
+  };
+
+  const handleAcknowledgementChange = (_event: any, checked: boolean) => {
+    setIsMultusAcknowledged(checked);
+  };
 
   return (
-    <>
-      <Checkbox
-        isChecked={networkType === NetworkType.MULTUS}
-        onChange={() =>
-          setNetworkType(
-            networkType === NetworkType.DEFAULT
-              ? NetworkType.MULTUS
-              : NetworkType.DEFAULT
-          )
-        }
-        label={t('Isolate network using Multus')}
-        description={t(
-          'Multus allows a network seperation between the data operations and the control plane operations.'
+    <div className="odf-multus-configuration">
+      {/* Network Section */}
+      <h2 className="odf-section-header">{t('Network')}</h2>
+      <Alert
+        data-test="odf-default-network-alert"
+        className="odf-alert odf-mb-md"
+        title={t('Data Foundation will use the default pod network')}
+        variant={AlertVariant.info}
+        isInline
+      >
+        {t(
+          'If you require a custom network configuration, you can modify the network settings after deployment.'
         )}
-        id="multus-checkbox"
-      />
-      {networkType === NetworkType.MULTUS && (
-        <MultusDropdown
-          setNetwork={setNetwork}
-          clusterNetwork={clusterNetwork}
-          publicNetwork={publicNetwork}
-          systemNamespace={systemNamespace}
+      </Alert>
+
+      {/* Isolate Network Section */}
+      <h2 className="odf-section-header odf-mt-lg">{t('Isolate Network')}</h2>
+      <div className="odf-indented-section">
```

Comment:
In clean code, intentional naming is about your domain (your business concepts, like "Multus"), not formatting details like "indentation".

```suggestion
      <div className="odf-multus-section">
```

Reply:
ack